env.d.ts

```diff
@@ -3,6 +3,7 @@
 interface ImportMetaEnv {
   readonly VITE_PORT: string;
   readonly VITE_LOCALHOST: string;
+  readonly VITE_MAPBOX_API_KEY: string;
```

Comment:
Wo wird der Key aktuell gespeichert? Sehe es in den Commits aktuell nicht

Reply:
Das ist gut, wenn du ihn nicht siehst. :) -> im Wiki unter Developer Guidelines habe ich erklärt, was man machen muss